localnet/path/config/shannon.example.yaml

```diff
@@ -0,0 +1,22 @@
+# To run a Local development environment using Shannon as the relaying protocol, copy this file and rename it to .config.yaml, then update
+# the `protocol.shannon_config.full_node_config.gateway_private_key` value with the correct value.
+# NOTE: only the gateway_private_key value should be updated.
+# shannon_config is required for using the shannon relayer in the Localnet environment.
+shannon_config:
+  full_node_config:
+    # do not change rpc_url
+    rpc_url: https://shannon-testnet-grove-seed-rpc.poktroll.com 
+    grpc_config:
+      # do not change host_port
```

Comment:
Ditto - make it obvious to new users but prevent experienced devs from asking questions